src/utils/sdkv2.ts

```diff
@@ -543,3 +544,26 @@ const getTokenBridgeToken = async (
 
   return config.tokens.get(tokenId);
 };
+
+export function isSameChainSwapEnabled(chain: Chain) {
```

Comment:
it would be better if the `Route` SDK interface had something like [`IS_SAME_CHAIN`](https://github.com/wormhole-foundation/wormhole-sdk-ts/blob/main/connect/src/routes/route.ts#L87) so we're not hard-coding logic around one route. in theory routes are supposed to be plugins.